src/ZEO/runzeo.py

```diff
@@ -342,6 +343,7 @@ def create_server(storages, options):
         storages,
         read_only = options.read_only,
         client_conflict_resolution=options.client_conflict_resolution,
+        msgpack=options.msgpack or os.environ.get('ZEO_MSGPACK'),
```

Comment:
IDK. ZConfig options handling is a mystery to me. I'll dig a little deeper.

Assuming that I can make it work the way you suggest, I will.